<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>382</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Nepomuk status</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="showAllCheckBox">
     <property name="text">
      <string>Show all services</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="servicesArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="servicesAreaContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>338</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="pluginLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>317</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
